src/main/java/ch/njol/skript/expressions/ExprHotbarSlot.java

```diff
@@ -96,19 +96,29 @@ public Slot get(Player player) {
 	@Nullable
 	public Class<?>[] acceptChange(ChangeMode mode) {
 		if (mode == ChangeMode.SET)
-			return new Class[] {Slot.class};
+			return new Class[] {Slot.class, Number.class};
 		return null;
 	}
 
 	@Override
 	public void change(Event event, @Nullable Object[] delta, ChangeMode mode) {
 		assert delta != null;
-		Slot slot = (Slot) delta[0];
-		if (!(slot instanceof InventorySlot))
-			return; // Only inventory slots can be hotbar slots
+		Object object = delta[0];
+		Number number = null;
+		if (object instanceof Slot) {
+			Slot slot = (Slot) delta[0];
+			if (!(slot instanceof InventorySlot))
```

Comment:
What's the point of having two instance checks? Can't you just replace the outer one with this?